apps/website/src/composables/wallet.ts

```diff
@@ -1,47 +1,82 @@
 import { ref } from 'vue'
 import { ethers } from 'ethers'
-import { WalletProvider } from '@/interfaces/WalletProvider'
+import useIopay from '@/composables/iopay'
+import useEthers from '@/composables/ethers'
+
+const { requestEthersAccount } = useEthers()
+const { getIoPayAccounts, sendIoPayTransaction } = useIopay()
 
 const defaultProviders = {
-  metamask: undefined,
-  coinbase: undefined,
+  MetaMask: undefined,
+  CoinbaseWallet: undefined,
 }
 
 export default function useWallet() {
   const ethereum: any = window.ethereum
-  const availableProviders = ref<Record<string, WalletProvider>>(
-    getAvailableProviders(ethereum)
-  )
-  const selectedProvider = ref<WalletProvider>({})
+  const availableProviders = ref<any>(getAvailableProviders(ethereum)) // TODO: Retype this?
```

Comment:
Try something like:

Create a new interface `BrowserProviders`:

```ts
import { EthersProvider } from '@/interfaces/EthersProvider'

export interface BrowserProviders {
    MetaMask?: EthersProvider
    CoinbaseWallet?: EthersProvider
}
```

Rename the method here to `getBrowserProviders` and use `BrowserProviders` instead of `any`.